android/app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,7 @@
       android:icon="@mipmap/ic_launcher"
       android:roundIcon="@mipmap/ic_launcher_round"
       android:allowBackup="false"
+      android:usesCleartextTraffic="true"
```

Comment:
This new flag added and set to true caught my attention. I believe that we should only be working with HTTPS files so it should not be needed. @patogallaiovlabs what are your thoughts?

Reply:
Flag removed in #707 which will be part of the same release (2.0.0) as this PR. All good!